src/web/mjs/connectors/MangaNel.mjs

```diff
@@ -101,9 +77,9 @@ export default class MangaNel extends Connector {
          * TODO: only perform requests when from download manager
          * or when from browser for preview and selected chapter matches
          */
-        this.requestOptions.headers.set('x-referer', payload.referer);
+        this.requestOptions.headers.set('x-referer', this.url + '/');
```

Comment:
Cherry picking : Its working, but a cleaner way to do that is new URL(this.url).href .
Do what you want anyway , no biggie.